tests/cylc-diff/01-icp.t

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2017 NIWA
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#-------------------------------------------------------------------------------
+# Test for "cylc diff --icp=CYCLE_POINT".
+. "$(dirname "$0")/test_header"
+
+set_test_number 3
+
+init_suite "${TEST_NAME_BASE}-1" <<'__SUITE_RC__'
+[cylc]
+    UTC mode = True
+[scheduling]
+    [[dependencies]]
```

Comment:
We could provide one of these suite defs with an ICP for a more comprehensive test.